app/release-notes/use-release-notes.ts

```diff
@@ -1,29 +1,46 @@
-import {getRepo} from "app/core/utils/get-repo"
-import {useEffect, useState} from "react"
+import {metaClient} from "app/ipc/meta"
+import {useEffect, useState, version} from "react"
 
-export function useReleaseNotes(version) {
+async function fetchNotes() {
+  const repo = await metaClient.repo()
+
+  const url = `https://api.github.com/repos/${repo}/releases/tags/v${version}`
+
+  try {
+    const resp = await fetch(url)
+    if (resp.ok) {
+      const json = await resp.json()
+      return json.body
+    } else {
+      throw new Error("Not found " + url)
+    }
```

Comment:
From MDN's documentation on [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API):

> The Promise returned from fetch() won’t reject on HTTP error status even if the response is an HTTP 404 or 500. Instead, it will resolve normally (with ok status set to false), and it will only reject on network failure or if anything prevented the request from completing

You should throw a more generic error if the stats code isn't 404.